### PR TITLE
adding v2.0.24

### PR DESCRIPTION
**Pull request by lisa-pangolin**

### CODE REVIEW

File: src/json/brace-data-exchange.schema.json [hunk 1]
```diff
@@ -1636,6 +1667,31 @@
...
           "description": "The property has problems that are environmental in nature (e.g. black mold.)"
         },
+        "has_new_principal_residence_over_100mi_away": {
+          "$ref": "#/definitions/tribool",
+          "title": "Has New Principal Residence Over 100 Miles Away",
+          "description": "The borrower or co-borrower has established a new principal residence more than 100 miles from the borrower’s current, or former, principal residence. "
+        },
+        "has_new_dependents_property_doesnt_meet_needs": {
+          "$ref": "#/definitions/tribool",
+          "title": "Has New Dependents property Doesn't Meet Needs",
```
Comment (MSGoodman) [inline src/json/brace-data-exchange.schema.json:1677]: This is super nitpicky but could we capitalize `property` here?

File: src/json/brace-data-exchange.schema.json [hunk 2]
```diff
@@ -1636,6 +1667,31 @@
...
+          "description": "Borrower to provide satisfactory evidence that there has been an increase in legal dependents and the principal residence fails to meet family needs. "
+        },
+        "is_borrower_vacating_residence": {
+          "$ref": "#/definitions/tribool",
+          "title": "Is Borrower Vacating Residence",
+          "description": "The borrower or co-borrower is vacating the principal residence with no intent to return. The principal residence will remain occupied by at least one existing borrower.  "
```
Comment (MSGoodman) [inline src/json/brace-data-exchange.schema.json:1683]: This is _also_ super nitpicky, but could we get rid of the trailing whitespace in all of the descriptions?

File: json/latest/brace-data-exchange.schema.json [hunk 3]
```diff
@@ -773,6 +773,37 @@
...
+        "caivrs_eligible":
+        {
+          "$ref": "#/definitions/tribool",
+          "title": "CAIVRS Eligibility",
+          "description": "One or more borrowers appear on the Credit Alert Verification Reporting System."
+        },
+        "has_been_borrower_on_loan_with_prior_fha_claim_within_3yrs": {
+          "$ref": "#/definitions/tribool",
+          "title": "Has Borrower Been Borrower On Loan With Prior FHA Claim Within 3 Years",
+          "description": "One or more borrowers, exception through inheritance, or co-signers on the subject property have had a prior loan on which a FHA partial claim has been paid within the previous three years from evaluation."
```
Comment (MSGoodman) [inline json/latest/brace-data-exchange.schema.json:785]: This is probably a product thing but perhaps this copy might read a bit better as something like

> One or more borrowers or co-signers on the subject property, except through inheritance, have had a prior loan on which a FHA partial claim has been paid within the previous three years.

File: src/json/brace-data-exchange.schema.json (hunk 2 above)
Comment (lisa-pangolin) [reply src/json/brace-data-exchange.schema.json]: Done.

File: src/json/brace-data-exchange.schema.json (hunk 1 above)
Comment (lisa-pangolin) [reply src/json/brace-data-exchange.schema.json]: Done.

**Review by kewpiedoll99 — APPROVED** [review]
LGTM

**Pull request merged**